src/home_work3/api/suggestionRoutes.js

```diff
@@ -0,0 +1,30 @@
+let express = require("express");
```

Comment:
Consider defaulting to usage of "const" wherever possible.
One of the cases - imported stuff from third-party libs. In general, you're not going to reassign these variable, so it makes sense to use "const" with them.

Reply:
Resolved as per the comment